crates/docs/src/static/index.html

```diff
@@ -55,6 +55,11 @@
                 <!-- Search Type Ahead -->
             </ul>
         </form>
+        <div id="llm-prompt-container">
+            <a id="llm-prompt-link" href="llms.txt">
+                🤖 LLM Prompt 🤖
```

Comment:
Based on the screenshots I think the two robots add a little too much visual noise.
```suggestion
                LLM docs
```
Could you add a nice hover text for this link as well?